pkg/model/components/etcdmanager/model.go

```diff
@@ -394,7 +394,7 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster *kops.EtcdClusterSpec) (*v1.Po
 			config.VolumeProvider = "do"
 
 			config.VolumeTag = []string{
-				fmt.Sprintf("kubernetes.io/cluster/%s=owned", b.Cluster.Name),
+				fmt.Sprintf("kubernetes.io/cluster=%s", b.Cluster.Name),
```

Comment:
Hmmm... can we follow the pattern that we use elsewhere (the `kubernetes.io/cluster/<clustername>` key).  I'm assuming that keys are unique in DO also, and the reason is that the `kubernetes.io/cluster=<clustername>` pattern does not allow for shared objects.  Admittedly sharing a volume would be weird, but sharing e.g. VPCs on AWS is pretty common.

This probably isn't as important on DO today, but it's unimportant until we find the first object we want to share :-)

Reply:
Sure @justinsb - this change would in-turn need a change to etcd-manager repo as well. May be we can get this merged if you are Ok, and then I'll take this separately to update both KOPS and etcd-manager repo. Please suggest if this sounds Ok for you.

Reply:
Sure ... that works :-)